js/compiler_test.ts

```diff
@@ -17,16 +16,25 @@ interface ModuleInfo {
   sourceMap: string | undefined;
 }
 
-const compilerInstance = DenoCompiler.instance();
+enum ScriptKind {
+  Unknown = 0,
+  JS = 1,
+  JSX = 2,
+  TS = 3,
+  TSX = 4,
+  External = 5,
+  JSON = 6,
+  Deferred = 7
+}
```

Comment:
We also have the MediaType in msg.fbs which doesn't correspond to this...
https://github.com/kitsonk/deno/blob/f4b01547cdad30ad22975aa4badddb726b94d092/src/msg.fbs#L123-L128
I'm not sure why this one is in compiler_test.ts either?

Reply:
Oh, I see - this is replicated from typescript - and you're not able to import it. Got it.